### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## Proposed Changes

Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

Note: I've filed some issues and made some PRs on some of the pkg_ items changed below, so they may technically not be incorrect today, but I believe that the changes should be made both here and upstream.

There are a number of benefits of using correct spelling, one is that it makes it easier to read, another is that it makes it easier to search/cross reference, another is that it reduces ambiguity/confusion.

I'm not claiming to be an erlang expert/user (we're investigating using RabbitMQ for a system, and I like to test the waters of projects to get a sense of how healthy they are with small changes). I haven't yet set up an erlang environment. So I haven't run the tests yet (I can, and understand that the contribution guide encourages me to do so).

### CODE REVIEW

File: src/amqp_selective_consumer.erl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-%% The contents of this file are subject to the Mozilla Public Licensbe
+%% The contents of this file are subject to the Mozilla Public License
```
Comment (jsoref) [inline src/amqp_selective_consumer.erl]: This is a change to a license blob, but in principle, it's correct.

File: test/system_SUITE.erl [hunk 2]
```diff
@@ -687,7 +687,7 @@ queue_unbind(Config) ->
 
 %% -------------------------------------------------------------------
 
-%% This is designed to exercize the internal queuing mechanism
+%% This is designed to exercise the internal queuing mechanism
```
Comment (jsoref) [inline test/system_SUITE.erl]: rabbiqmq is en-GB, -ize is a en-US ending, but even so, this is still not correct in en-US...

Comment (michaelklishin) [reply test/system_SUITE.erl]: Sticking to the UK spelling is becoming increasingly harder as the majority of our team are based outside of the UK now. We don't feel particularly strongly either way :)

**Comment by michaelklishin** [comment]
Thank you!

File: src/amqp_selective_consumer.erl (hunk 1 above)
Comment (michaelklishin) [reply src/amqp_selective_consumer.erl]: Yes, that's a good catch and we can accept it 👍

File: test/system_SUITE.erl (hunk 2 above)
Comment (jsoref) [reply test/system_SUITE.erl]: Yeah, I'm familiar w/ all sides of the problem.

If you're feeling adventurous, you could try adding a Travis hook to improve/retain spelling behaviors (or behaviours, up to you...). I've provided a hook for two projects, one has held the line w/ it and seems happy, one ran away screaming after a week or so.

**Pull request merged**